extensions/canvasMedical/actions/index.ts

```diff
@@ -0,0 +1,15 @@
+import { updatePatient } from './updatePatient'
+import { createPatient } from './createPatient'
+import { updateAppointment } from './updateAppointment'
+import { createAppointment } from './createAppointment'
+import { createTask } from './createTask'
+import { extractPatientInfo } from './extractPatientFields'
+
+export const actions = {
+  updatePatient,
+  createPatient,
+  updateAppointment,
+  createAppointment,
+  createTask,
+  extractPatientInfo,
+}
```

Comment:
Only setter actions (create, update) should be exported, no getters (read/get), so it's ok to include `create  questionnaire response`  but not the read action for these responses

`extractPatientInfo` should also NOT be exported, since that is only used with the JSON type data points, which we don't support yet